pages/admin/lessons.tsx

```diff
@@ -4,11 +4,18 @@ import { AdminLessonsSideBar } from '../../components/admin/lessons/AdminLessons
 import { withGetApp, GetAppProps } from '../../graphql'
 import { Lesson } from '../../graphql/index'
 import { AdminLayout } from '../../components/admin/AdminLayout'
-
+import Error, { StatusCode } from '../../components/Error'
+import LoadingSpinner from '../../components/LoadingSpinner'
 const Lessons: React.FC<GetAppProps> = ({ data }) => {
+  const { loading, error, lessons } = data
+  if (loading) return <LoadingSpinner />
```

Comment:
This is unnecessary, it is handled in AdminLayout
https://github.com/garageScript/c0d3-app/blob/master/components/admin/AdminLayout.tsx#L16

There error is handled in AdminLayout as well, except one difference is that there is no `message` prop value passed into the Error component that is returned

Reply:
Fixed.